### PR TITLE
[IMPROVE] Spanish home translation

### PR DESCRIPTION
**Pull request by nYCSTs**
Co-authored-by: Arthur-Gaudium<arthurmatos@gaudium.com.br>

## Proposed changes (including videos or screenshots)

Update translation of home page to Spanish. The page had some untranslated texts.
image

### CODE REVIEW

File: apps/meteor/packages/rocketchat-i18n/i18n/es.i18n.json [hunk 1]
```diff
@@ -2334,6 +2340,7 @@
   "Invitation_Subject_Default": "Se te ha invitado a [Site_Name]",
   "Invite": "Invitación",
   "Invites": "Invitaciones",
+  "Invite_and_add_members_to_this_workspace_to_start_communicating": "Invitar y agregar miembros este espacio de trabajo para comenzar a comunicarse.",
```
Comment (KevLehman) [inline apps/meteor/packages/rocketchat-i18n/i18n/es.i18n.json:2343]:
```suggestion
  "Invite_and_add_members_to_this_workspace_to_start_communicating": "Invite y agregue miembros a este espacio de trabajo para comenzar a comunicarse.",
```

File: apps/meteor/packages/rocketchat-i18n/i18n/es.i18n.json [hunk 2]
```diff
@@ -2375,6 +2382,7 @@
   "Join_default_channels": "Unirse a los canales por defecto",
   "Join_the_Community": "Unirse a la Comunidad",
   "Join_the_given_channel": "Unirse al canal dado",
+  "Join_rooms": "Unirse a salas",
```
Comment (KevLehman) [inline apps/meteor/packages/rocketchat-i18n/i18n/es.i18n.json:2385]: Únase a un canal

File: apps/meteor/packages/rocketchat-i18n/i18n/es.i18n.json [hunk 3]
```diff
@@ -4106,6 +4119,7 @@
   "Tag_removed": "Etiqueta eliminada",
   "Tag_already_exists": "La etiqueta ya existe",
   "Take_it": "Atiéndelo",
+  "Take_rocket_chat_with_you_with_mobile_applications": "Llévate Rocket.Chat contigo con aplicaciones móviles.",
```
Comment (KevLehman) [inline apps/meteor/packages/rocketchat-i18n/i18n/es.i18n.json:4122]:
```suggestion
  "Take_rocket_chat_with_you_with_mobile_applications": "Llévate Rocket.Chat contigo con nuestras aplicaciones móviles.",
```

File: apps/meteor/packages/rocketchat-i18n/i18n/es.i18n.json [hunk 4]
```diff
@@ -2257,6 +2262,7 @@
   "Instructions_to_your_visitor_fill_the_form_to_send_a_message": "Instrucciones para que el visitante rellene el formulario para enviar un mensaje",
   "Insert_Contact_Name": "Insertar nombre del contacto",
   "Insert_Placeholder": "Insertar marcador de posición",
+  "Install_rocket_chat_on_your_preferred_desktop_platform": "Instale Rocket.Chat en su plataforma de escritorio preferida.",
```
Comment (KevLehman) [inline apps/meteor/packages/rocketchat-i18n/i18n/es.i18n.json:2265]:
```suggestion
  "Install_rocket_chat_on_your_preferred_desktop_platform": "Instale Rocket.Chat en su sistema operativo de preferencia.",
```

**Review by KevLehman — CHANGES_REQUESTED** [review]
Even when these are not "exact" translations, they help the text to read more human. Wdyt?

**Comment by nYCSTs** [comment]
> Even when these are not "exact" translations, they help the text to read more human. Wdyt?

100% agree with your suggestions. ty

**Review by KevLehman — APPROVED** [review]

**Pull request merged**